Source/dom/models/Asset.js

```diff
@@ -0,0 +1,126 @@
+import { isNativeObject } from 'util'
+import { WrappedObject, DefinedPropertiesKey } from '../WrappedObject'
+import { Types } from '../enums'
+import { Factory } from '../Factory'
+import { Color, colorToString } from '../style/Color'
+import { Gradient } from '../style/Gradient'
+import { isWrappedObject } from '../utils'
+
+export class Asset extends WrappedObject {}
+
```

Comment:
let's do `delete Asset[DefinedPropertiesKey].id` here instead of below
```suggestion
Asset[DefinedPropertiesKey] = { ...WrappedObject[DefinedPropertiesKey] }
delete Asset[DefinedPropertiesKey].id

```